types/argon2-browser/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for argon2-browser 1.1
+// Project: https://github.com/antelle/argon2-browser#readme
+// Definitions by: Ivan Gabriele <https://github.com/ivangabriele>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export namespace argon2 {
```

Comment:
Seems wierd for a library to expose itself via a single export, but that's apparently what it does.

Reply:
Indeed...